ui-speedspacechart/src/components/helpers/drawElements/speedLimitTags.ts

```diff
@@ -160,18 +136,62 @@ export const drawSpeedLimitTags = async ({
             ) {
               tooltip = {
                 cursorX: cursor.x + MARGIN_LEFT,
-                cursorY: Y_POSITION,
+                cursorY: cursor.y,
                 text: 'Incompatible with the infrastructure',
```

Comment:
```suggestion
                text: 'Incompatible with the infrastructure',
```
we should begin to add those texts in the user translations object prop, then keep this trad as a placeholder